dnf/cli/commands/__init__.py

```diff
@@ -472,7 +472,7 @@ def run_on_repo(self):
     class ReinstallOldSubCommand(SubCommand):
         """Implementation of the reinstall-old sub-command."""
 
-        aliases = ('reinstall-old',)
+        aliases = ('reinstall-old', 'ri-old', 'ri-o',)
```

Comment:
please only one, but if you change reinstall, please change it according to it.

Reply:
I suggest using `ri-old` here. Less ambiguous.